src/metabase/models/table.clj

```diff
@@ -264,3 +265,23 @@
   [{:keys [db_id] :as table}]
   (-> (serdes.base/load-xform-basics table)
       (assoc :db_id (db/select-one-field :id 'Database :name db_id))))
+
+(defmethod serdes.base/storage-path "Table" [table _]
```

Comment:
```suggestion
(defmethod serdes.base/storage-path "Table" [table _context]
```

It makes the code a lot more readable IMO if you do something like `_context` instead of just `_` so we know what the unused variable is at a glance

Reply:
Done.